executor/src/witgen/vm_processor.rs

```diff
@@ -82,26 +78,19 @@ impl<'a, T: FieldElement> VmProcessor<'a, T> {
         self.data
     }
 
-    fn last_row(&self) -> DegreeType {
-        self.fixed_data.degree - 1
-    }
-
+    /// Starting out with a single row, iteratively append rows until we have degree + 1 rows
+    /// (i.e., we have the first row twice).
```

Comment:
This will make more sense after #674 when the processor no longer has access to all rows, but only the current block. In that case, it can't properly handle wrapping by itself, but blocks have to be stitched together from the outside.

Similar to the block machine's `Processor`, it always returns the first row of the next block, so that the caller can merge blocks.